build.gradle.kts

```diff
@@ -20,6 +20,7 @@ configurations {
 dependencies {
   annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
 
+  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:1.3.0")
```

Comment:
mine is set to 
`implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:2.0.0")`

Reply:
Removed this as not relevant as we are not upgrading SQS. We are still on Spring Boot 2 on SLM and hence we have to use 1.3.0